guardian/support-dotcom-components
# Enable self-served contribution types in banner

## What does this change?
This PR represents the last part of the work to create a seamless journey for users who engage with a marketing message on the main site (epic or banner) which includes an amounts card, to make sure the amounts they see in the supporter+ landing page are the same as they saw in the banner or epic message.

The last outstanding tasks are to:
+ Port the new amounts card functionality, using the v3 amounts model in AWS, over to the moment template and designable banner components
+ Update the banner amounts cards to respect the contributionType data set (via RRCP) in the selected amounts test variant



## How to test
+ All existing Storybook stories which include an amounts card should not change.
+ Compare storybook stories against a live banner with an amounts card to make sure its display mirrors the live banner responsively, at all screen widths
+ Create additional Storybook stories to check for amounts data edge case displays (eg: no amounts values in the test variant, etc)

## Images
TODO: add images of new Storybook stories so reviewers know what they're looking at/for
